### PR TITLE
Feature/sem 023

### PR DESCRIPTION
**Pull request by ELVISrodrigo123**
feature/SEM-023
https://ditmar.github.io/backoffice-openscience/branch-feature/SEM-023/?path=src/ui/atoms/TabLabel

### CODE REVIEW

**Comment by ELVISrodrigo123** [comment]
https://ditmar.github.io/backoffice-openscience/branch-feature/SEM-023/?path=src/ui/atoms/TabLabel

File: src/ui/atoms/TabLabel/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,49 @@
+$--ads-colors-blue-60: #1163fe;
+$--ads-colors-gray-10: #ccc;
+$--ads-colors-gray-20: #e3e3e3;
+$--ads-colors-gray-40: #b0b0b0;
+$--ads-colors-gray-100: #000;
```
Comment (Ditmar) [inline src/ui/atoms/TabLabel/styles.module.scss:5]: move the variants color to the globals folder

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "typescript.tsdk": "node_modules\\typescript\\lib"
```
Comment (Ditmar) [inline .vscode/settings.json:2]: why you need to do this change here

**Review by Ditmar — CHANGES_REQUESTED** [review]
read the comments

**Pull request closed without merging**